src/parser/jobs/rdm/modules/Gauge.js

```diff
@@ -314,6 +341,28 @@ export default class Gauge extends Module {
 					<Trans id="rdm.gauge.suggestions.black-mana-invuln-why">You lost {this._blackManaLostToInvulnerable} Black Mana due to misses or spells that targeted an invulnerable target</Trans>
 				</Fragment>,
 			}))
+
+			this.statistics.add(new SimpleStatistic({
```

Comment:
I'd be inclined to merge these into a single statistic, given their related nature. Not necessarily add the values together, just display both in the same box.